vcd/resource_vcd_org_vdc.go

```diff
@@ -382,6 +418,24 @@ func setOrgVdcData(d *schema.ResourceData, vcdClient *VCDClient, adminOrg *govcd
 	if err := d.Set("metadata", getMetadataStruct(metadata.MetadataEntry)); err != nil {
 		return fmt.Errorf("error setting metadata: %s", err)
 	}
+
+	if vcdClient.Client.APIVCDMaxVersionIs(">= 33.0") {
+		assignedVmSizingPolicies, err := adminVdc.GetAllAssignedVdcComputePolicies(nil)
+		if err != nil {
+			log.Printf("[DEBUG] Unable to get assigned VM sizing policies")
+			return fmt.Errorf("unable to get assigned VM sizing policies %s", err)
+		}
+		var policyIds []string
+		for _, policy := range assignedVmSizingPolicies {
+			policyIds = append(policyIds, policy.VdcComputePolicy.ID)
+		}
+		vmSizingPoliciesSlice := convertToTypeSet(policyIds)
+		vmSizingPoliciesSet := schema.NewSet(schema.HashSchema(&schema.Schema{Type: schema.TypeString}), vmSizingPoliciesSlice)
+		_ = d.Set("vm_sizing_policy_ids", vmSizingPoliciesSet)
```

Comment:
We should probably handle error for complex type

Reply:
ty. added error handling